cmd/up/forwards.go

```diff
@@ -129,6 +112,67 @@ func (up *upContext) sshForwards(ctx context.Context) error {
 	return nil
 }
 
+func addToForwarder(up *upContext) error {
+	ticker := time.NewTicker(1 * time.Second)
+	to := time.NewTicker(up.Dev.Timeout.Resources)
+	var forwardErr error
```

Comment:
need to store the error if the timeout finish we need to return a proper error to the user